src/main/scala/scorex/core/network/NodeViewSynchronizer.scala

```diff
@@ -150,7 +150,7 @@ MR <: MempoolReader[TX]](networkControllerRef: ActorRef,
   def sendExtension(remote: ConnectedPeer,
                     status: HistoryComparisonResult.Value,
                     extOpt: Option[Seq[(ModifierTypeId, ModifierId)]]): Unit = extOpt match {
-    case None => log.warn(s"extOpt is empty for: $remote. Its status is: $status.")
+    case None => log.warn(s"extOpt is empty for: $remote. Its status is: $status." : String)
```

Comment:
For some strange reason the Scala compiler infers a Serialisable (or similar) type instead of String. I didn't research on the reason for this, but I can check if I find a more elegant solution.